action.yml

```diff
@@ -5,5 +5,5 @@ inputs:
     description: 'The GITHUB_TOKEN secret'
     required: true
 runs:
-  using: 'node12'
+  using: 'node16'
```

Comment:
node 16? or 18?

Reply:
Github actions don't support node16.  You'll see node18 in the commit history :)